apps/mull-ui/src/app/pages/profile/add-friends/search-users.tsx

```diff
@@ -0,0 +1,69 @@
+import { faSearch, faUserPlus } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Dialog } from '@material-ui/core';
+import React, { useState } from 'react';
+import { useFriendsQuery } from '../../../../generated/graphql';
+import { CustomTextInput, MullBackButton, TopNavBar } from '../../../components';
+import ContactRow from '../../../components/contact-row/contact-row';
+import './add-friends.scss';
+
+export interface SearchUsersPageProps {
+  open: boolean;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const SearchUsersPage = ({ open, setOpen }: SearchUsersPageProps) => {
+  const [searchValue, setSearchValue] = useState('');
+
+  // TODO: Replace the test query by the actual strangers query
+  const { data, loading } = useFriendsQuery();
+
+  if (loading) return <div className="page-container">Loading...</div>;
+
+  // TODO: Goes with the first TODO - filter by strangers
+  const contactRows = data.friends
+    .filter(({ name }) => name.toLowerCase().includes(searchValue.toLowerCase()))
+    .map(({ latestPost, ...friend }, index) => (
+      <ContactRow
+        key={'contact-row-' + index}
+        userId={friend.id}
+        userName={friend.name}
+        lastMessage={latestPost ? latestPost.message : ''}
+        icon={faUserPlus}
+      />
+    ));
+
+  return (
+    <Dialog fullScreen transitionDuration={5} open={open}>
```

Comment:
⚠So from what I can see this is basically a fullscreen modal that mimics a normal page that is overlayed on top of the page when the user clicks on the search bar.

I don't think this is a good idea. I know you did this to follow the mockup but the solution has quite a bit of duplication which is never good. 

There are multiple ways of handling this. My suggestion would be to disregard the mockup, remove the top bar, replace the "Add Friend" title with something like "Search for new Friends", and have the back button say "Cancel Search" or something similar.

This would fix another issue I have with the search page, which is the **confusion** I get when I search for stuff. Since the page is basically unchanged, I have the impression that I'm searching through the list of users that have either added or been added by me, not new users.

I made these changes quickly on your branch so you can see how it would look. can be adjusted obviously
![image](https://user-images.githubusercontent.com/23544999/113453456-ff90e300-93d3-11eb-825a-f8779b192149.png)